.gitignore

```diff
@@ -15,3 +15,23 @@
 npm-debug.log
 yarn-error.log
 ###< symfony/webpack-encore-bundle ###
+
+###> symfony/phpunit-bridge ###
+.phpunit.result.cache
+/phpunit.xml
+###< symfony/phpunit-bridge ###
+
+###> phpunit/phpunit ###
+/phpunit.xml
+.phpunit.result.cache
+###< phpunit/phpunit ###
+
+###< phpcsfixer/phpcsfixer ###
+"/.php-cs-fixer.php",
```

Comment:
The quotes quand the final comma must be remove (same the line below).

Reply:
Actually it looks like this whole block can be removed, they is already the config for PHP CS below.